html/js/q-doc.js

```diff
@@ -131,6 +131,13 @@ QDoc.postProcessDoc = function(docJson) {
             element.throws[tCount].description = S(element.throws[tCount].description).humanize();
         }
 
+        for(var sCount = 0; sCount < element.see.length; sCount++) {
+            element.see[sCount] = {
+                func: S(element.see[sCount]).humanize(),
```

Comment:
You should remove the call to humanize here. For functions with camel-casing (e.g. `.qdoc.getCwd`) is renders as `.qdoc.get cwd` which is incorrect.